dsf-mpi/dsf-mpi-client/src/main/java/org/highmed/mpi/client/MasterPatientIndexClientFactory.java

```diff
@@ -0,0 +1,12 @@
+package org.highmed.mpi.client;
+
+import java.util.Properties;
+
+import org.springframework.core.env.Environment;
+
+public interface MasterPatientIndexClientFactory
+{
+	MasterPatientIndexClient getWebserviceClient(Environment environment);
```

Comment:
```suggestion
	MasterPatientIndexClient createMasterPatientIndexClient(Function<String, String> propertyResolver);
```

See MasterPatientIndexClientPdqFactory comment.